Sources/AWSLambdaEvents/APIGateway+V2+Encodable.swift

```diff
@@ -0,0 +1,50 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftAWSLambdaRuntime open source project
+//
+// Copyright (c) 2017-2022 Apple Inc. and the SwiftAWSLambdaRuntime project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftAWSLambdaRuntime project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import HTTPTypes
+
+#if canImport(FoundationEssentials)
+import class FoundationEssentials.JSONEncoder
+import struct FoundationEssentials.Data
+#else
+import class Foundation.JSONEncoder
+import struct Foundation.Data
+#endif
+
+public enum APIGatewayResponseError: Error {
+    case failedToEncodeBody(Error)
+}
+
+extension APIGatewayV2Response {
```

Comment:
If you're doing this for APIGatewayV2 is it worthwhile doing it for APIGateway

Reply:
done